esridump/dumper.py

```diff
@@ -169,6 +169,9 @@ def _find_oid_field_name(self, metadata):
             for f in metadata['fields']:
                 if f.get('type') == 'esriFieldTypeOID':
                     oid_field_name = f['name']
+                elif f['alias'].lower() == 'objectid':
+                    oid_field_name = f['alias']
```

Comment:
I think you might want `f['name']` in there. The alias can be any alphanumeric characters whereas the name usually is the database column name.

Reply:
@iandees just updated, good catch, thank you! added commit below